google/cloud/firestore_v1/base_vector_query.py

```diff
@@ -69,6 +71,11 @@ def _to_protobuf(self) -> query.StructuredQuery:
         else:
             raise ValueError("Invalid distance_measure")
 
+        # Coerce ints to floats as required by the protobuf.
+        distance_threshold_proto = None
+        if self._distance_threshold:
+            distance_threshold_proto = float(self._distance_threshold)
```

Comment:
If self._distance_threshold is 0, distance_threshold_proto will be None. Is that expected?

Reply:
No thanks for the catch.